features/received_command.feature

```diff
@@ -66,7 +66,7 @@ Feature:  received_command
     Given a deployed rule:
       """
       rule 'Execute rule when member of group receives any command' do
-        received_command AlarmModes.items
+        received_command AlarmModes.members
```

Comment:
noet that this change is because the test is wrong. it only _used_ to pass out of sheer dumb luck. the global `items` method is leaking onto every object (which I've mentioned might be a possibility before, but now that I've seen it happen I'm more motivated to fix), so this was setting up a trigger on every openhab object, _not_ members of a group. and the internal method separating non-groups from groups was short-circuiting on array-like objects, instead of only GroupMembers objects. but since Items is now an Enumerable, instead of delegating to an Array, it was missing that and blowing up because it wasn't a GroupMembers object either (see below where that method gets simplified)

Reply:
It would be neater if this change is moved into #279

Reply:
Done.